src/app/ApplicationTemplate.Shared.Views/Styles/Controls/Frame.xaml

```diff
@@ -1,15 +1,15 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:android="http://uno.ui/android"
-                    xmlns:ios="http://uno.ui/ios"
-                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-                    xmlns:uic="using:Uno.UI.Controls"
-                    xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:xamarin="http://uno.ui/xamarin"
-                    mc:Ignorable="ios android xamarin">
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:android="http://uno.ui/android"
+					xmlns:ios="http://uno.ui/ios"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:uic="using:Uno.UI.Controls"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:xamarin="http://uno.ui/xamarin"
```

Comment:
Since we don't use Xamarin anymore, not_win might be a better usage of the name, both do the same thing
https://platform.uno/docs/articles/platform-specific-xaml.html#available-prefixes